### PR TITLE
Add TailwindCSS

### LINKED ISSUE #611: Begin adopting tailwind
Make tailwind available in dev workspace and leverage TW classes in one component.

### PR DESCRIPTION
**Pull request by ItsMeBrianD**
### Description

This builds on the work started in #623, adding TailwindCSS and styling the `BigValue` and `ErrorChart` components.

Resolves #611 

Additions from the previous PR:
 - TailwindCSS was reinitialized and reconfigured using [`svelte-add`](https://github.com/svelte-add/tailwindcss)
 - Tailwind escapes were hoisted into the theme where appropriate (e.g. text shadow, grid template items)

### Checklist
- [ ] For UI or styling changes, I have added a screenshot or gif showing before & after
- [ ] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

### Noteworthy Items for Review:
 - I used `data-viz` for style scoping for an svg height that is nested in a 3rd party library. I am not sure if this style is expected elsewhere; and the `:global` may be eligible for hoisting into `app.css`.

### Before
image

### After
image

### CODE REVIEW

**Comment by mcrascal** [comment]
@ItsMeBrianD I am not fussed about the awkward whitespace on the error state for now. 

It looks like the sparkline [here](https://evidence-development-workspace-git-feat-tai-2a826b-evidence-dev.vercel.app/text-and-metrics/big-value/) isn't rendering quite right. [Edit, nevermind it looks like this is actually regression in main.]

**Comment by mcrascal** [comment]
Fooling around with this locally. 



**Alerts:** 

Nits: 
* Check rendering in Safari. 
* I'd drop the icon (sorry). I think these will be more flexible without it, and the alignment gets a bit jumbled when you put multiple lines in the slots. 
* I'd cut padding to p-2, rest of the page is fairly dense 
* I'd drop the borders (sorry) 

Sticky mode: 

I can get two behaviours with the sticky prop:

1. `sticky = {true}` and the alert will follow the scroll past it's position on the page
2. `{sticky}` in reference to the bool on the example page, and I get a full-width 

* When it's in "sticky mode", and is scrolling with the header, I would remove the rounding on the corners. 
* If you are going full-width, I would have no margin on either side, currently there's a gap to the left and not on the right (see image). If that's a pain, I'd just stay in the article width. 
* If you're going full-width, I'd remove margin bottom while scrolling so that multiple alerts can stack. 

CleanShot 2023-03-22 at 18 02 02@2x

**Tabs:**

These are sweet -- I'd go sans serif on the labels, and switch to the blue we use for active pages in the sidebar as the "active" colour over the green

**Comment by hughess** [comment]
These are great!

One thing worth considering is the syntax for true/false props - currently, the rest of the component library accepts those props as strings and converts them to boolean (idea being that it keeps the prop syntax cleaner w/ less curly braces)

File: sites/example-project/src/components/viz/BigValue.svelte [hunk 1]
```diff
@@ -120,66 +121,21 @@
...
-            <span class="comparison-type">{comparisonTitle}</span>
+            <span class="text-grey-700 font-normal">{comparisonTitle}</span>
         </p> 
     {/if}
     {/if}
 </div>  
 
 <style>
-    :global(.sparkline svg) {
+    /*
```
Comment (Winterhart) [inline sites/example-project/src/components/viz/BigValue.svelte:134]: 👍🏼

File: sites/example-project/src/components/ui/Portal.svelte [hunk 2]
```diff
@@ -0,0 +1,91 @@
+<script context="module">
```
[the remark this answers is not in the document]
Comment (ItsMeBrianD) [reply sites/example-project/src/components/ui/Portal.svelte:1]: This was originally written for the `Alert` component; when pulling it out of the content context to escape margins. We took that functionality out but it is still a useful component to have so I just left it in. (There are modifications that I made after copying it from `svelte-portal`)

File: sites/example-project/src/routes/+layout.svelte [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<script>
```
[the remark this answers is not in the document]
Comment (ItsMeBrianD) [reply sites/example-project/src/routes/+layout.svelte:1]: This is autogenerated by `svelte-add`, we can move this to the layout in pages if that's preferred

File: packages/preprocess/tailwind.config.cjs [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    extend: {
+      textShadow: {
+        DEFAULT: '1px solid var(--tw-shadow-color)'
+      },
+      gridTemplateRows: {
+        auto: "auto"
+      },
+      fontFamily: {
+        monospace: [
+          "Menlo",
```
[the remark this answers is not in the document]
Comment (ItsMeBrianD) [reply packages/preprocess/tailwind.config.cjs:16]: Agreed, changes like that will get easier the more we get migrated to tailwind

File: sites/example-project/src/components/ui/Alert.svelte [hunk 5]
```diff
@@ -0,0 +1,45 @@
+<script>
+  // Based on the alert from FlowBite: https://flowbite.com/docs/components/alerts/
```
[the remark this answers is not in the document]
Comment (ItsMeBrianD) [reply sites/example-project/src/components/ui/Alert.svelte:2]: I just used Flowbite as a starting point; it seemed like a safer bet than trying to come up with a design

File: sites/example-project/src/routes/+layout.svelte (hunk 3 above)
[the remark this answers is not in the document]
Comment (Winterhart) [reply sites/example-project/src/routes/+layout.svelte:1]: Let's remove it we have a `+layout.svelte` in `pages` folder

File: sites/example-project/src/components/ui/Alert.svelte (hunk 5 above)
[the remark this answers is not in the document]
Comment (Winterhart) [reply sites/example-project/src/components/ui/Alert.svelte:2]: As, long as we respect their `Licensing` policies I saw some `Pricing` on `FlowBite`

[the remark this answers is not in the document]
Comment (ItsMeBrianD) [reply sites/example-project/src/components/ui/Alert.svelte:2]: Pricing is for their page layouts and figma files, the components themselves are free to use

[the remark this answers is not in the document]
Comment (ItsMeBrianD) [reply sites/example-project/src/components/ui/Alert.svelte:2]: To follow up on this now that I'm back at a keyboard; you can see the pricing [here](https://flowbite.com/pro/#pricing), this falls under the `Community Edition`, which lets us use the components, just not the blocks or page layouts

**Review by Winterhart — APPROVED** [review]

**Comment by mcrascal** [comment]
Looking great to me. I'm OK to skip the "unrounding". Not sure what's going on with the MacOS 16 test, but once passing I think you're good to go.

**Comment by ItsMeBrianD** [comment]
I'm not entirely sure where to start with the MacOS / 16 test, it looks like an error getting credentials for the BigQuery tests, this has come up frequently on PRs I've worked on

**Pull request merged**